cpp/src/slice2cpp/CPlusPlusUtil.cpp

```diff
@@ -43,6 +43,26 @@ toOptional(const string& s)
     return "Ice::optional<" + s + ">";
 }
 
+bool
+isOptionalProxyOrClass(const TypePtr& type)
+{
+    auto optional = OptionalPtr::dynamicCast(type);
```

Comment:
Missed a comment in my review I guess!

We could probably make this:
```
if (auto optional = OptionalPtr::dynamicCast(type))
```
Since we don't need it anywhere else except inside the scope of the if anyways.

Reply:
Fixed.